src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/BlockPhysicsListener.java

```diff
@@ -84,6 +84,7 @@ public void onPistonRetract(BlockPistonRetractEvent e) {
     @EventHandler(ignoreCancelled = true)
     public void onLiquidFlow(BlockFromToEvent e) {
         Block block = e.getToBlock();
+        Material type = block.getType();
 
         // Check if this Material can be destroyed by fluids
         if (SlimefunTag.FLUID_SENSITIVE_MATERIALS.isTagged(block.getType())) {
```

Comment:
or you forgot to replace block.getType() here, either one works

Reply:
```suggestion
        if (SlimefunTag.FLUID_SENSITIVE_MATERIALS.isTagged(type)) {
```